langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibXMLTest.java

```diff
@@ -281,6 +284,36 @@ public void testSelectingTextFromXml() {
         BRunUtil.invoke(compileResult, "testSelectingTextFromXml");
     }
 
+    @Test
+    public void testGetDescendants() {
+        String[] expectedDescendants = {"<CD>\n" +
```

Comment:
Shall we add more tests, which include other xml types, (comments and processing instructions).
Also shall we try to assert them in the Ballerina itself.

Reply:
Added and refactored in 2b6c68b820bb0c13ce59393ce873213c381a4224